bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -151,6 +152,7 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
 
     {% if task.depends_on_fivetran != None -%}
     {% for fivetran_task in task.depends_on_fivetran -%}
+    {% if fivetran_task not in fivetran_seen -%}
```

Comment:
fixes bug where fivetran_tasks would be defined mulitple times if referenced multiple times, by following the same pattern used for external dependencies